src/server/controllers/usersCtrl.js

```diff
@@ -2,34 +2,23 @@ const db = require('../db/connection.js');
 
 // Signup User
 exports.addUser = (req, res) => {
-  db.none('insert into users(name, username, password, email, weight, bmi, goal, points)' +
-      'values(${name}, ${username}, ${password}, ${email}, ${weight}, ${bmi}, ${goal}, ${points})',
+  console.log('req.body in addUser: ', req.body);
+  db.none('insert into users(name, username, password, email, weight, bmi, goal)' +
+      'values(${name}, ${username}, ${password}, ${email}, ${weight}, ${bmi}, ${goal})',
     req.body)
     .then(() => {
       res.status(201)
         .json({
           status: 'success',
-          message: 'Inserted user into db'
+          message: 'Inserted user into db',
+          data: req.body
```

Comment:
- i just sent the whole user object back but user_id shouldn't be necessary, all we need is the username to fetch info (as long as we make the username field unique), especially when we use external apis